DISCOURSE.md

```diff
@@ -0,0 +1,42 @@
+# How to run loginbot for Discourse?
+
+The loginbot implements the OAuth2 specification partially and it covers the API [Discourse](https://www.discourse.org/) needs. Therefore, you can use it for 
+adding "Login with DeltaChat" to your Discourse instance. 
+
+## Getting a compiled version
+
+Each release comes with a compiled version for Linux x86 64 musl. This binary is stripped, optimized and should run on any Linux server with the
+said architecture regardless of the `libc` used in the distribution as the binary comes with its own `libc`. However, if you want it for some other 
+platform, you can [install the Rust toolchain](https://www.rust-lang.org/learn/get-started) and then `cargo build -r` in the project's directory
+to get an optimized release binary for your platform.
+
+## Prerequirements
+
+ - A server to run loginbot on it. Root access is NOT required nor it is recommended to run loginbot as root.
+ - A webserver like nginx to act as reverse proxy so that the outside world can talk with loginbot web APIs.
+ - An email account for the loginbot
+ - Admin access to the Discourse instance
+
+## Steps
+
+ - Download the latest release from Releases and get the binary
+ - Modify the TOML config file to your need. You need an email account somewhere for the loginbot
```

Comment:
> the TOML config file to your need.

user might not know what file you mean, better also mention the path where to find it.

Like I would write something like this:

After you have downloaded the library, download the example config file from "address" (maybe provide a wget/curl command that downloads it and also renames it?) rename it to `config.toml` put your credentials into it and move it next to the binary.